RecoBTag/DeepFlavour/plugins/DeepFlavourTagInfoProducer.cc

```diff
@@ -300,12 +305,13 @@ void DeepFlavourTagInfoProducer::produce(edm::Event& iEvent, const edm::EventSet
       auto entry = c_sortedindices.at(i);
       // get cached track info
       auto & trackinfo = trackinfos.at(i);
+      if(flip_ && (trackinfo.getTrackSip3dSig() > 10.0)){continue;}
```

Comment:
shouldn't this be configurable like in ```pfDeepCSVNegativeTagInfos.computer.trackSelection.sip3dSigMax``` and related parameters?

Reply:
Yes it should be. In fact it should always be the same as pfDeepCSVNegativeTagInfos.computer.trackSelection.sip3dSigMax since deepflavour inherits some tagging variables from deepcsv. If its not the same there will be a mismatch in cuts between some of deep flavour tagging variables.

Reply:
OK.
A more strict solution is to use the provenance of the pfDeepCSVNegativeTagInfos input.

```c++
    const edm::Provenance *prov = shallow_tag_infos.provenance();
    edm::ParameterSet psetFromProvenance = edm::parameterSet(*prov);
//get the parameter values here
```

Reply:
Done in the new PR at https://github.com/cms-sw/cmssw/pull/23467/files#diff-778e7825bed1fc0ad81083affff99fe3R277